scripts/dev/clang-format.sh

```diff
@@ -93,10 +93,10 @@ for p in "$@"; do
     if [[ -d "$p" ]]; then
         :
     elif [[ -f "$p" ]]; then
-        if [[ "$p" =~ .*\.[ch] ]]; then
+        if [[ "$p" =~ .*\.([ch]|h\.in) ]]; then
```

Comment:
`(c|h|in)`  (drop the `[ch]` part)

Reply:
I think you want a literal $ at the end of the pattern to match the end of the string

Reply:
you are correct.  so  replace ` .*\.([ch]|h\.in)` with ` .*\.(c|h|h\.in)$`